### ISSUE TITLE
ApplicationError

### ISSUE THREAD

**Issue by anmingyu11** [GitbookIO/gitbook#2173]
This doesn't happen often, but it looks like something is broken. Hitting the back button and trying again might be your best bet. If that doesn't work you can head back to the homepage.

There might be more information on our status page which is reporting All Systems Operational.

what the hell is this ? keep doing this???

**Comment by benxie** [comment]
i'm planning to host my own gitbook but gitbook-cli doesn work neither

**Comment by benxie** [comment]
@anmingyu11  i dont know, i guess they are focusing on the https://www.gitbook.com/, a product that people are paying real money with. but the thing is, with such a bad quality of legacy.gitbook.com, I won't have much confidence buying the product that they are selling as a customer. :(

**Comment by anmingyu11** [comment]
@benxie  yeah, to be honest the new www.gitbook.com is more hard to use than the legacy, i am still confused how to manage that, and in their index do not even have a guide or something, and i email them 4 month ago ,and nothing changed .

**Comment by anmingyu11** [comment]
@benxie  it has been fixed, legacy is recoverd, just tell you that
